tasks/coverage/minifier_test262.snap

```diff
@@ -1,3 +1,4 @@
 minifier_test262 Summary:
 AST Parsed     : 44738/44738 (100.00%)
-Positive Passed: 44738/44738 (100.00%)
+Positive Passed: 44737/44738 (100.00%)
+Expect to Parse: "language/punctuators/S7.7_A1.js"
```

Comment:
`1 < 2 > 3` will be compressed to “!0>3” while it shouldn't be folded(“1<2>3”). Not found how to do it yet.

[ESBuild result](https://esbuild.github.io/try/#dAAwLjE4LjQALS1taW5pZnkAMSA8IDIgPiAz)